src/test/java/edu/tamu/app/service/manager/GitHubServiceTest.java

```diff
@@ -423,6 +443,7 @@ public void testGetGitHubInstanceByToken() throws IOException {
     public void testGetCardsWithNote() throws Exception {
         when(TEST_CARD1.getContent()).thenReturn(null);
         List<Sprint> sprints = gitHubService.getAdditionalActiveSprints();
+        System.out.println("\n\n\nsprints: " + sprints.get(1).getCards().get(0).getId() + "\n\n\n");
```

Comment:
I don't think this log is needed.

Reply:
Good catch, thanks. I've removed the log statement.